pt-BR/api_app.md

```diff
@@ -1,15 +1,15 @@
 **NÃO LEIA ESTE ARQUIVO NO GITHUB, OS GUIAS SÃO PUBLICADOS NO https://guiarails.com.br.**
 **DO NOT READ THIS FILE ON GITHUB, GUIDES ARE PUBLISHED ON https://guides.rubyonrails.org.**
 
-Using Rails for API-only Applications
+Usando Rails para aplicações API
```

Comment:
```suggestion
Usando Rails para aplicações somente API
```

Reply:
Concordo, Tomás! Eu até pensei nisso, só não coloquei porque as contribuições anteriores usavam API sem o somente kkkk vou colocar! Acho que é bom deixar isso especifico pelo menos no início pra quem ta lendo o artigo.